issue templates (#299)
SuperDARN/rst: .github/ISSUE_TEMPLATE/feature_template.md

# Requesting New Feature

**Name**: *Feature name*

**function**:  *filename.c* 

**bin/lib**: */directory*

## Scope

*Create a check list using markdowns [checklist syntax](https://help.github.com/en/github/managing-your-work-on-github/about-task-lists) on the scope of the feature.*

*What will the feature do?*

*What will the feature not do?* 

## Description

> *Please provide a description of the new feature and if will improve or solve a problem.*

Reviewer: ```suggestion
*Please provide a description of the new feature, and whether it will improve/solve a problem.*
```

### RST Checklist 

-  [ ] Does this fit within RST's scope? 
-  [ ] Is this a minor change?
-  [ ] Is this a major change?
-  [ ] Do you need help developing it? 
-  [ ] Have you created a RST project (github project) to reflect the process of developing it? 

### Developement help

> *RST does not have a dedicated development team on hand to full feature development. 

Reviewer: ```suggestion
*RST does not have a dedicated development team for full feature development. 
```

However, if you are able to provide some time and help, the community will try to aid you the best way they can in making your feature possible.*

*If you need help please indicate what expertise you are looking for:*
- [ ] developer help with design and integration of code
- [ ] scientific help with ensuring what you develop is scientifically correct/accurate
- [ ] project management to help organize what steps need to be taken to make this possible 

> *Please also provide any other information you may need help with not including testing/reviewers of code*

Reviewer: ```suggestion
*Please also provide any other information you may need help with (not including testing/reviewers of code)*
```
